apps/condo/domains/banking/schema/BankAccountReportTask.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Generated by `createschema banking.BankAccountReportTask 'account:Relationship:BankAccount:CASCADE; organization:Relationship:Organization:CASCADE; status:Select:processing,completed,cancelled,errored; progress:Integer; meta:Json;'`
+ */
+
+const { Relationship, Integer, Select } = require('@keystonejs/fields')
+const { values } = require('lodash')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/banking/access/BankAccountReportTask')
+const { generateReportsTask } = require('@condo/domains/banking/tasks/generateReports')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+
+const { BANK_SYNC_TASK_STATUS } = require('../constants')
+
+
+const BankAccountReportTask = new GQLListSchema('BankAccountReportTask', {
+    schemaDoc: 'Information about the report generation process',
+    fields: {
+
+        account: {
+            schemaDoc: 'Bank account for which the current report generation operation is running.',
+            type: Relationship,
+            ref: 'BankAccount',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            access: {
+                read: true,
+                create: true,
+                update: false,
+            },
+        },
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        status: {
+            schemaDoc: 'Status of current generation operation',
+            type: Select,
+            options: values(BANK_SYNC_TASK_STATUS),
+            defaultValue: BANK_SYNC_TASK_STATUS.PROCESSING,
+            isRequired: true,
+        },
+
+        progress: {
```

Comment:
Optional you can restrict values for `progress` field to be in interval (0…100) if you already have a strategy to calculate progress.